src/renderer/components/dialogs/MessageDetail/ReadReceipts.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import { BackendRemote, onDCEvent } from '../../../backend-com'
+import { T } from '@deltachat/jsonrpc-client'
+import { selectedAccountId } from '../../../ScreenController'
+
+import styles from './styles.module.scss'
+import { Avatar } from '../../Avatar'
+import { InlineVerifiedIcon } from '../../VerifiedIcon'
+import moment from 'moment'
+import useTranslationFunction from '../../../hooks/useTranslationFunction'
+type ReadReceiptsListProps = { messageId: number }
+
+export function ReadReceiptsList(props: ReadReceiptsListProps) {
+  const tx = useTranslationFunction()
+  const accountId = selectedAccountId()
+  const [receipts, setReceipts] = useState<T.MessageReadReceipt[]>([])
+  useEffect(() => {
+    const update = () => {
+      BackendRemote.rpc
+        .getMessageReadReceipts(accountId, props.messageId)
+        .then(setReceipts)
+    }
+    update()
+    return onDCEvent(accountId, 'MsgRead', ({ msgId }) => {
+      if (msgId === props.messageId) {
+        update()
+      }
+    })
+  }, [props.messageId, accountId])
+
+  if (receipts.length === 0) {
+    return null
+  }
+  return (
+    <div className={styles.ReadReceiptsContainer}>
+      <div className={styles.Heading}>
+        <div className={styles.DoubleCheckmarkIcon}></div>
+        {tx('pref_read_by')}
+      </div>
+      <table className={styles.ReadReceiptBox}>
+        {receipts.map(receipt => (
+          <ReadReceipt receipt={receipt} />
+        ))}
+      </table>
```

Comment:
This is not a valid table though, it doesn't have rows and columns just `<div>`s. If you don't want to get too deep, `<ul>` with `<li>`s would also be fine.